src/css/clustermanagement.css

```diff
@@ -247,6 +247,23 @@ text.icon-any {
     font-size: 56px;
 }
 
+.text-btn {
```

Comment:
What's the point of this? Isn't it better to just use a link?

Reply:
This is how it is in the repository create view. I left it like that for consistency of look.